feat(managers/ocb): add new manager for OpenTelemetryCollectorBuilder (#26509)
renovatebot/renovate: docs/usage/configuration-options.md

## bumpVersion

> Currently, this setting supports following managers only:

Reviewer: ```suggestion
Currently, this config option only works with these managers:
```

- `helmv3`
- `npm`
- `nuget`
- `maven`
- `ocb`
- `pep621`
- `poetry`
- `sbt`

> Raise a feature request if you have a use for it with other package managers.

Reviewer: ```suggestion
Raise a feature request if you want to use this config option with other package managers.
```

> Its purpose is if you want Renovate to update the `version` field within your package file any time it updates dependencies within.
> Usually this is for automatic release purposes, so that you don't need to add another step after Renovate before you can release a new version.

Reviewer: ```suggestion
Use `bumpVersion` if you want Renovate to update the `version` field in your package file when it updates the dependencies in that file.
This can be handy when you have automated your package's release, as you you don't need extra steps after the Renovate upgrade, you can just release a new version.
```

You're not editing this text with this PR, but I think we can improve the text. How about this?

## bumpVersion

> Currently, this setting supports `helmv3`, `npm`, `nuget`, `maven`, `ocb`, `pep621`, `poetry` and `sbt` only, so raise a feature request if you have a use for it with other package managers.

Reviewer 2: We shold convert this to a list. @HonkingGoose WDYT?

Reviewer: Let's put this in a list. 🙂
